src/modules/user/user.service.ts

```diff
@@ -59,7 +59,8 @@ export class UserService {
       const { email, fId, role } = request['user'];
       const userExists = await this.userModel.findOne({ email: email }).lean();
       if (userExists) {
-        throw new ConflictException('User already exists');
+        console.log('User Already Exists');
+        return;
```

Comment:
change this

Reply:
The issue has been solved by creating a user if it does not exist in the middleware. This solves the problem of login, logout and relogin and also prevents duplicacy by caching and this flow is also used in most of the Mern apps using firebase authentication